lib/omnibus-ctl.rb

```diff
@@ -506,6 +508,45 @@ def reconfigure(exit_on_success=true)
       end
     end
 
+    def check_license_acceptance(override_accept = false)
+      license_guard_file_path = File.join(etc_path, ".lic.accepted")
```

Comment:
Should this potentially live in /var/opt instead?  We are trying to move away from having state files like this in etc.